src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/blocks/include/index.test.js

```diff
@@ -369,7 +252,6 @@ describe('convertInclude', () => {
       model: {
         href: 'idt2',
         type: 'idt2',
-        html: null,
```

Comment:
Sorry I missed this @rhenshaw56, thanks for suggesting this change: https://github.com/bbc/simorgh/pull/6873/files#diff-ace5f927a44cc0bc96a151479cba4bbaR119-R120